### PR TITLE
chore: setup linkinator for custom applications

### PR DESCRIPTION
**Pull request by davifantasia**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,6 +43,8 @@
     "template-starter:start": "yarn --cwd application-templates/starter start",
     "template-starter:start:prod:local": "yarn --cwd application-templates/starter start:prod:local",
     "build": "NODE_ENV=production lerna run build --no-private",
+    "postbuild": "yarn check-links",
```
Comment (emmenko) [inline package.json:46]: This should only be used as post build of the website.

```suggestion
    "postbuild:website": "yarn check-links",
```

File: package.json [hunk 2]
```diff
@@ -43,6 +43,8 @@
...
+    "check-links": "npx linkinator ./website/public/custom-applications/public",
```
Comment (emmenko) [inline package.json:47]:
```suggestion
    "check-links": "npx linkinator ./website/public/custom-applications",
```

File: package.json (hunk 1 above)
Comment (emmenko) [reply package.json:46]: Better to include it in the postbuild hook of the website package

**Review by emmenko — CHANGES_REQUESTED** [review]
Thanks. Can you move it to the `website` folder only though?
It does not need to run in the main repo.

File: website/linkinator.config.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+  "recurse": true,
+  "silent": true,
+  "skip": [
+    ".(json|js|png|jpg|svg|css)(\\?(.*)+)?$",
```
Comment (emmenko) [inline website/linkinator.config.json:5]: I included this rule to exclude check for static assets (which are handled by Gatsby anyway).

**Comment by adnasa** [comment]
This came out of nowhere for me.
What's beneficial with linkinator?

**Comment by emmenko** [comment]
It's what we use in the docs websites to check that links are correct.

**Review by emmenko — APPROVED** [review]

**Pull request merged**